python/gink/__main__.py

```diff
@@ -9,6 +9,8 @@
 from importlib import import_module
 from os import environ
 from json import dumps
+import datetime # allows setting of datetime objects from CLI
+# TODO: don't import all of datetime
```

Comment:
A container contents dump with datetime objects will look something like datetime.datetime(3, 12, 14, ...), so using `from datetime import datetime` won't work for loading. This is something to figure out, but maybe can be figured out in a separate PR.